### ISSUE TITLE
Need ttnn support for batch_norm1d

### ISSUE THREAD

**Issue by punithsekar** [tenstorrent/tt-metal#21113]
For pointpillars model, Need batchnorm1D operation in TT framework, [Torch link](https://docs.pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html).

Input shapes: [4352,64,64]

Operation: BatchNorm1d(64, eps=0.001, momentum=0.01, affine=True, track_running_stats=True)

**Comment by punithsekar** [comment]
CC: @saichandax

**Comment by saichandax** [comment]
@punithsekar , Please add link to torch source

**Comment by punithsekar** [comment]
Updated it in the description!

**Comment by vguduruTT** [comment]
- ttnn Batch_norm1d support is needed For DETR3D Model ([#21560](https://github.com/tenstorrent/tt-metal/issues/21560))

- Corresponding torch operation - `BatchNorm1d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)`
- Input shapes - [1, 256, 1024], [8, 256, 128]

**Comment by vguduruTT** [comment]
cc: @saichandax

**Comment by zzigler-tt** [comment]
@dvartaniansTT can you please review and re-assign or close out if this is not needed. TY
